src/openrct2/ride/Vehicle.h

```diff
@@ -378,6 +378,24 @@ struct Vehicle : EntityBase
 
     void Loc6DCE02(const Ride& curRide);
     void Loc6DCDE4(const Ride& curRide);
+    /**
+     * @return true If UpdateTrackMotion() must return after calling this function.
```

Comment:
Can we make these static functions and not member functions

Reply:
No. They directly change properties and call into other private methods.